pkg/utils/clusterinfo/fetcher.go

```diff
@@ -376,3 +376,34 @@ func getPDNodesHealth(pdEndPoint string, httpClient *http.Client) (map[string]st
 	}
 	return memberHealth, nil
 }
+
+// GetAlertCountByAddress receives alert manager's address like "ip:port", and it returns the
+//  alert number of the alert manager.
+func GetAlertCountByAddress(address string, httpClient *http.Client) (int, error) {
+	ip, port, err := parseHostAndPortFromAddress(address)
+	if err != nil {
+		return 0, err
+	}
+
+	apiAddress := fmt.Sprintf("http://%s:%d/api/v2/alerts", ip, port)
```

Comment:
why not specify address use %s directly?

Reply:
I think here needs checkings, else the response will be ambigious.